lib/graphql/execution/interpreter/runtime/graphql_result.rb

```diff
@@ -157,6 +157,11 @@ def fix_result_order
               end
             end
           end
+
+          # hook for breadth-first implementations to signal when collecting results.
+          def collect_result(result_name, result_value)
```

Comment:
`?` for predicate method?

```suggestion
          def collect_result?(result_name, result_value)
```

Reply:
I assume you skipped it because it can have side effects. Probably better as is 👍